src/electionguard/api/load_ballots.c

```diff
@@ -0,0 +1,139 @@
+#include <stdlib.h>
+
+#include <electionguard/api/load_ballots.h>
+
+#include "api/base_hash.h"
+#include "directory.h"
+#include "api/filename.h"
+
+static API_LoadBallots_status initialize_coordinator(uint32_t num_selections);
+static API_LoadBallots_status load_ballots(uint64_t start_index, 
+                    uint64_t count,
+                    uint32_t num_selections,
+                    char *import_filepath,
+                    char **out_external_identifiers,
+                    struct register_ballot_message *out_encrypted_ballots);
+
+// Global state
+static Voting_Coordinator load_coordinator = NULL;
```

Comment:
What naming convention are we using? This variable name can easily be mistaken for a local variable.

Reply:
added an underscore until #62 is implemented